docs/config/README.md

```diff
@@ -807,12 +807,12 @@ it would have been `nixpkgs/pkgs`.
 | `truncation_symbol` | `""`                                               | The symbol to prefix to truncated paths. eg: "…/"                                |
 | `repo_root_style`   | `None`                                             | The style for the root of the git repo when `truncate_to_repo` option is set to false.|
 | `home_symbol`       | `"~"`                                              | The symbol indicating home directory.                                           |
+| `convert_slash`     | `true`                                             | Convert slash on output on Windows                                               |
 
 <details>
 <summary>This module has a few advanced configuration options that control how the directory is displayed.</summary>
 
-| Advanced Option             | Default | Description                                                                                                                                                            |
-| --------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Advanced Opt-------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
It looks like you might have accidentally deleted the rest of this line, and then the editor took over?